VikingQuiz.UI/src/Components/MyQuizesPage/DisplayPlayedGamesComponent/DisplayPlayedGamesComponent.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import './DisplayPlayedGamesComponent.css'
+import HttpService from '../../../services/HttpService';
+
+interface IState {
+    games: any[]
+    gameData: IGameData[],
+    noGamesPlayed: boolean,
+    sortedGameDataByDate: any[]
+}
+
+interface IGameData {
+    numberOfPlayers: number,
+    gameDate: string,
+    quizTitle: string,
+    quizPhoto: string
+}
+
+class DisplayPlayedGamesComponent extends React.Component<any, IState>{
+    
+    private readonly apiAddressForGames = 'http://localhost:60151/api/game/usergames';
```

Comment:
Instead of http://localhost:60151/ as a hardcoded string maybe used the  apiUrl from the constants.tsx file.  This will allow the component to work when it will be deployed because then the url will be different.